### PR TITLE
Update some of zoggy's packages urls

### PR DESCRIPTION
**Pull request by sagotch**

### CODE REVIEW

**Comment by sagotch** [comment]
`md5` does not seem to be correct

```
$ md5sum chamo-release-2.03.tar.gz 
621cb304a7f6201fc118ae22ec22c3c3  chamo-release-2.03.tar.gz
```

File: packages/chamo/chamo.2.01/opam [hunk 1]
```diff
@@ -18,6 +18,6 @@ synopsis:
   "A source code editor, even if it can be used to edit any text file."
 extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
 url {
-  src: "http://zoggy.github.com/chamo/chamo-2.01.tar.gz"
+  src: "https://framagit.org/zoggy/chamo/-/archive/release-2.03/chamo-release-2.03.tar.gz"
```
Comment (mseri) [inline packages/chamo/chamo.2.01/opam:21]:
```suggestion
  src: "https://framagit.org/zoggy/chamo/-/archive/release-2.01/chamo-release-2.01.tar.gz"
```

**Comment by sagotch** [comment]
> We usually compare the content, if the release tarball contains the same code, then we update the url otherwise we use the cache

I'll check when I have time

File: packages/chamo/chamo.2.01/opam [hunk 2]
```diff
@@ -18,6 +18,6 @@ synopsis:
   "A source code editor, even if it can be used to edit any text file."
 extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
 url {
-  src: "http://zoggy.github.com/chamo/chamo-2.01.tar.gz"
+  src: "https://framagit.org/zoggy/chamo/-/archive/release-2.01/chamo-release-2.01.tar.gz"
   checksum: "md5=e9e2047c929680884c35cee6a973735c"
```
Comment (mseri) [inline packages/chamo/chamo.2.01/opam:22]:
```suggestion
  checksum: "md5=3a9285847aa9bf09bcebfab7a7e8c758"
```

File: packages/chamo/chamo.2.02/opam [hunk 3]
```diff
@@ -22,6 +22,6 @@ synopsis:
   "A source code editor, even if it can be used to edit any text file."
 extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
 url {
-  src: "http://zoggy.github.com/chamo/chamo-2.02.tar.gz"
+  src: "https://framagit.org/zoggy/chamo/-/archive/release-2.02/chamo-release-2.02.tar.gz"
   checksum: "md5=efedc2453f1ca3e24c9de93f236db578"
```
Comment (mseri) [inline packages/chamo/chamo.2.02/opam:26]:
```suggestion
  checksum: "md5=8a9b3bd35abfcd4db7b8f2910f48395b"
```

File: packages/chamo/chamo.2.03/opam [hunk 4]
```diff
@@ -28,6 +28,6 @@ synopsis:
   "A source code editor, even if it can be used to edit any text file."
 extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
 url {
-  src: "http://zoggy.github.com/chamo/chamo-2.03.tar.gz"
+  src: "https://framagit.org/zoggy/chamo/-/archive/release-2.03/chamo-release-2.03.tar.gz"
   checksum: "md5=f2dc1e57c4205349796a8745065f5e95"
```
Comment (mseri) [inline packages/chamo/chamo.2.03/opam:32]:
```suggestion
  checksum: "md5=621cb304a7f6201fc118ae22ec22c3c3"
```

**Comment by mseri** [comment]
I manually compared the packages with the old cache via pkgdiff, they are unchanged

File: packages/chamo/chamo.2.02/opam [hunk 5]
```diff
@@ -22,6 +22,6 @@ synopsis:
   "A source code editor, even if it can be used to edit any text file."
 extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
```
Comment (mseri) [inline packages/chamo/chamo.2.02/opam:23]:
```suggestion
license: "LGPL-3.0-only"
homepage: "https://framagit.org/zoggy/chamo"
extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
```

File: packages/chamo/chamo.2.01/opam [hunk 6]
```diff
@@ -18,6 +18,6 @@ synopsis:
   "A source code editor, even if it can be used to edit any text file."
 extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
```
Comment (mseri) [inline packages/chamo/chamo.2.01/opam:19]:
```suggestion
license: "LGPL-3.0-only"
homepage: "https://framagit.org/zoggy/chamo"
extra-files: ["chamo.install" "md5=61635a19e58f6af4d4587471c79bef4a"]
```

File: packages/chamo/chamo.2.01/opam [hunk 7]
```diff
@@ -16,8 +16,12 @@ depends: [
 install: [make "install"]
 synopsis:
   "A source code editor, even if it can be used to edit any text file."
+homepage: "https://zoggy.frama.io/chamo/"
+license: "LGPL-3.0-only"
+doc: "https://zoggy.frama.io/chamo/"
+dev-repo: "https://framagit.org/zoggy/chamo.git"
```
Comment (mseri) [inline packages/chamo/chamo.2.01/opam:22]:
```suggestion
dev-repo: "git+https://framagit.org/zoggy/chamo.git"
```

File: packages/chamo/chamo.2.02/opam [hunk 8]
```diff
@@ -20,8 +20,12 @@ depends: [
 install: [make "install-lib"]
 synopsis:
   "A source code editor, even if it can be used to edit any text file."
+homepage: "https://zoggy.frama.io/chamo/"
+license: "LGPL-3.0-only"
+doc: "https://zoggy.frama.io/chamo/"
+dev-repo: "https://framagit.org/zoggy/chamo.git"
```
Comment (mseri) [inline packages/chamo/chamo.2.02/opam:26]:
```suggestion
dev-repo: "git+https://framagit.org/zoggy/chamo.git"
```

File: packages/chamo/chamo.2.03/opam [hunk 9]
```diff
@@ -1,9 +1,10 @@
 opam-version: "2.0"
 maintainer: "zoggy@bat8.org"
 authors: ["Maxence Guesdon"]
-homepage: "http://zoggy.github.io/chamo/"
+homepage: "https://zoggy.frama.io/chamo/"
 license: "LGPL-3.0-only"
-doc: ["http://zoggy.github.io/chamo/doc.html"]
+doc: "https://zoggy.frama.io/chamo/"
+dev-repo: "https://framagit.org/zoggy/chamo.git"
```
Comment (mseri) [inline packages/chamo/chamo.2.03/opam:7]:
```suggestion
dev-repo: "git+https://framagit.org/zoggy/chamo.git"
```

**Comment by mseri** [comment]
Thanks @sagotch

**Pull request merged**